_posts/2022-12-21-trino-summit-2022-astronomer-recap.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: "Using Trino with Apache Airflow for (almost) all your data problems"
+author: "Philippe Gagnon, Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/astronomer.jpg
+---
+
+As we close in on the final talks from [Trino Summit 2022]({% post_url 
+2022-11-21-trino-summit-2022-recap %}), this next talk dives into how to set up
+Trino for batch processing. Trino has historically been well-known for providing
+fast adhoc analytics queries as opposed to long-running, resource intensive
+queries commonly classified as batch or ETL queries. This is due to the fact
+that Trino kills queries that run out of resources in favor of lower latency.
+Earlier this year, Trino added
+[new features](https://trino.io/blog/2022/05/05/tardigrade-launch.html) that
+simplify running batch queries using a new fault-tolerance execution mode. This
```

Comment:
Suggest rewording to "Trino added features to better support batch queries with a new fault-tolerant execution mode" and make "fault tolerant execution mode" the link text